.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+# Rule definitions: http://flake8.pycqa.org/en/latest/user/error-codes.html
+# D203: 1 blank line required before class docstring
+# F401: 'identifier' imported but unused
+# E402: module level import not at top of file
+# C901: too complex
+# W503: line break before binary operator
+exclude = venv*,__pycache__,node_modules,bower_components
+ignore = D203,W503
+max-complexity = 10
+max-line-length = 120
+per-file-ignores =
+    **/__init__.py : F401
```

Comment:
Doesn't look like this rule is actually broken by the repo as it stands now, so we can drop this.

Reply:
This is just copied from dm-utils - I think we should have consistent rules across our repo's?

Reply:
We don't - we have rules specific to the repository, and in general try to have as few exclusions as possible.

Reply:
Hmm well specifically imports in init files are a common thing to do (the idea being to create a simplified API I think?) but ok I guess, personally I'd strongly prefer consistency.

Reply:
I think we're trying to be consistent with the idea of minimising exclusions as much as possible. This leads to different rules in each app, but they're consistently following the principle.

(I don't really think it's been given that much thought tbh, this conversation is probably about as much as has been had on this file, so take anything I say with a pinch of salt...)